cypress/integration/manualEntry.spec.ts

```diff
@@ -0,0 +1,25 @@
+describe("", function () {
+  it("should find a button to press to add source by user entry manually", function () {
```

Comment:
I realized that `cypress/integration/addSource.spec.ts` and `cypress/integration/manualEntry.spec.ts` are both testing the components on the same page (`/languages`), and I am wondering why did you create two different test files?

Reply:
Since they are for different components, like AddSource has the test addSource.spec.ts, ManualEntry has the test manualEntry.spec.ts. If we don't want tests for components, I can put these two together. I just think that we were testing components instead of the entire page, as we did before.

Reply:
Personally, I would say it is good to have just one since you only have two test cases for the component, but you can definitely make the call.